installer/charts/rhtap-infrastructure/templates/odf/backing-store.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.odf.enabled }}
+apiVersion: noobaa.io/v1alpha1
+kind: BackingStore
+metadata:
+  finalizers:
+  - noobaa.io/finalizer
+  labels:
+    app: noobaa
+  name: noobaa-pv-backing-store
+  namespace: {{ .Values.odf.namespace }}
+spec:
+  pvPool:
+    numVolumes: 1
+    resources:
+      limits:
+        cpu: 1000m
+        memory: 4000Mi
+      requests:
+        cpu: 800m
+        memory: 800Mi
+        storage: {{ .Values.odf.backingStorageSize }}
+    storageClass: gp3-csi 
```

Comment:
Need to remove this so that it picks up the default storage class, this happens to be the default, so functionally is working as expected

Reply:
This is done and verified